sssd_test_framework/roles/ipa.py

```diff
@@ -811,6 +1040,151 @@ def __get_member_args(self, members: list[IPAUser | IPAGroup | str]) -> list[str
         external = [x for item in members if isinstance(item, str) for x in ("--external", item)]
         return [*users, *groups, *external]
 
+    def idgroupoverride(self) -> IDGroupOverride:
+        """
+        Add override to the IPA Group.
+
+        .. code-block:: python
+            :caption: Example usage
+
+            @pytest.mark.topology(KnownTopology.IPA)
+            def test_example(client: Client, ipa: IPA):
+                ipa.idview("newview1").add(description="This is a new view")
+                ipa.idview("newview1").apply(hosts=f"{client.host.hostname}")
+                ipa.group("group-1").add().idgroupoverride().add_override("newview1", gid=1344567)
+                client.sssd.restart()
+                g_lookup = client.tools.getent.group("group-1")
+                assert g_lookup.gid == 1344567
+
+        :return: New IDOverride object.
+        :rtype: IDOverride
+        """
+        return IDGroupOverride(self)
+
+
+class IDGroupOverride(IPAGroup):
+    """
+    IPA group ID override
+    """
+
+    def __init__(self, group: IPAGroup) -> None:
+        """
+        :param user: IPA group object
+        :type user: IPAGroup
+        """
+        super().__init__(group.role, group.name)
+        self.name = group.name
+
+    def add_override(
+        self,
+        idview_name: str,
+        *,
+        description: str | None = None,
+        name: str | None = None,
+        gid: int | None = None,
+        **kwargs,
+    ) -> IDGroupOverride:
+        """
+        Add a new Group ID override.
+
+        :param idview_name: Name of IDView.
+        :type idview_name: str
+        :param description: Description of the IDView.
+        :type description: str | None, defaults to None.
+        :param name: Name of the IDView.
+        :type name: str | None, defaults to None.
+        :param gid: GID of the IDView.
```

Comment:
'Description', 'Name' and 'GID' of a group object being overridden, not of "the IDView".

Reply:
Updated.

Reply:
updated, please check again.

Reply:
As I wrote: `The same in all other places: user/group add/modify...`

For example - IDUserOverride::add_override -
```
param uid: User ID number of the IDView.
param gid: Group ID of the IDView.
...
```

Reply:
I updated everywhere now, but please do check again.

Reply:
@alexey-tikhonov, Would you like me to resolve this comment, or is there anything else you’d like to add or suggest?

Reply:
I didn't check but I trust other reviewers.